### PR TITLE
Make HUD react to selected units

### PR DESCRIPTION
**Pull request by Royber121**
My naive try to implement changing of the HUD when a ship is selected.

### CODE REVIEW

**Comment by LinuxDonald** [comment]
Thank you for the PR.
Are you done with it? Can it be reviewed or do you still work on it?

File: Assets/Player/PlayerHud.gd [hunk 1]
```diff
@@ -15,9 +17,25 @@ var _debug_messages = [
...
+func _on_selected():
+	var selected_units = get_tree().get_nodes_in_group("selected_units")
+	if selected_units.size() == 0:
+		# should in the future hide all unit-related HUD-Elements
+		tab_widget._set_detail_visibility(false)
+	elif selected_units.size() >= 1: 
+		tab_widget._change_on_select(selected_units)
+	
+		
+
```
Comment (aaronfranke) [inline Assets/Player/PlayerHud.gd]: 1. The style convention we are using for this project is to have 1 line between functions. Here you have 3.

2. Private functions (ones that start with an underscore that aren't *lifecycle* functions like _ready or _process) should go after public ones.

**Review by aaronfranke — CHANGES_REQUESTED** [review]

**Comment by Royber121** [comment]
Actually I'm still working on it. I will incorporate the requested changes.

**Comment by Royber121** [comment]
I just realized that artism was doing a lot of work on the GUI as well. I just had some free time and started to code, I hope I didnt't do something artism had in mind as well (and could do much better)

**Comment by Royber121** [comment]
I reverted develop and committed the changes to another branch HUD-updates-Unit-selection. I will close this PR and open another.

**Pull request closed without merging**